indexer/services/ender/__tests__/handlers/funding-handler.test.ts

```diff
@@ -167,6 +168,47 @@ describe('fundingHandler', () => {
     );
   });
 
+  it('successfully processes and clears cache for a new funding rate with non-existent market', async () => {
```

Comment:
nit: update test to indicate the funding event has both an existing and a non-existent market.

Reply:
done